index.html

```diff
@@ -27,7 +32,7 @@
         
         <div class="bio">
         <div class="content">
-            <h1 class="bio-header">I'm Josiah. Glad to see you!</h1>
+            <h1 class="bio-header">I'm Josiah. <br>Glad to see you!</h1>
             <p class="bio-text">I'm a software developer! I can help you build a product, feature or website. Look Through
             some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
             <div class="connect">
```

Comment:
please add some margin before the `let's connect` and `social icons` in your project so have a better design of your portfolio

Figma design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/951c086e-a9bc-4498-b6d9-ee8a2028262e)

Your design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/9cc6eba6-1b16-43fc-94f1-4c22d75d8192)